.dockerignore

```diff
@@ -0,0 +1,6 @@
+.git
+.github
+*.md
+.dockerignore
```

Comment:
I will forever lol at the dockerignore needing to ignore itself

Reply:
I agree it's a bit weird. It was modeled after gitignore which does have different design considerations but oh well.